Libraries/src/Amazon.Lambda.AppSyncEvents/Amazon.Lambda.AppSyncEvents.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\..\buildtools\common.props" />
+
+  <PropertyGroup>
+    <Description>Amazon Lambda .NET Core support - AWS AppSync package.</Description>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net8.0</TargetFrameworks>
+    <AssemblyTitle>Amazon.Lambda.AppSyncEvents</AssemblyTitle>
+    <Version>0.0.1</Version>
+    <AssemblyName>Amazon.Lambda.AppSyncEvents</AssemblyName>
+    <PackageId>Amazon.Lambda.AppSyncEvents</PackageId>
+    <PackageTags>AWS;Amazon;Lambda</PackageTags>
+  </PropertyGroup>
+
+	<PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
```

Comment:
No need for the condition if we only target net8.0

Reply:
Removed

Reply:
This condition should only be removed if the targets are net8 and higher. Right now, you are targeting both net6 and net8. You should drop net6.

Reply:
Now target framework is only net8.0